.github/workflows/master.yaml

```diff
@@ -7,18 +7,19 @@ jobs:
   tests:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions/setup-go@v2
-      with:
-        go-version: '^1.15'
-    - run: go version
-    - run: sudo apt-get install redis-server
-    - run: make deps
-    - run: make check-fmt
-    - run: make build
-    - run: make vet
-    - run: make staticcheck
-    - run: make check-race
-    - run: make cicheck
-    - run: make gosec
-    - run: make publish-coverage
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          # https://www.npmjs.com/package/semver#caret-ranges-123-025-004
+          go-version: '^1.15'
```

Comment:
Since 1.15 is no longer a supported go version it may make sense to also update it here, to further avoid confusion? Now that you change this anyway :)

Reply:
Done, thanks